nodejs/FatBeaconPeripheral/FatBeacon.js

```diff
@@ -50,5 +50,30 @@ AdvertisementData.makeUrlBuffer = function(name) {
 
 /*********************************************************/
 
-// Start Advertising name.
-eddystoneBeacon.advertiseUrl('New Fatbeacon');
+var webpageCharacteristic = require('./webpageCharacteristic');
+
+var webCharacteristic = new webpageCharacteristic();
+
+var service = new bleno.PrimaryService({
+  uuid: 'ae5946d4-e587-4ba8-b6a5-a97cca6affd3',
+  characteristics: [
+    webCharacteristic
+  ]
+});
+
+bleno.once('advertisingStart', function(err) {
+  if (err) {
+    throw err;
+  }
+
+  console.log('on -> advertisingStart');
+  bleno.setServices([
+    service
+  ]);
+});
+
+var data = new Buffer.from('Hello World of FatBeacon!');
```

Comment:
I think it would make sense for you to put this code right after you define the characteristic.  Otherwise, it gives the impression of a race condition...what happens if you get a write request before you've told it what to do on write requests?  I know nothing will be connected to the service yet, but this behavior should be defined before the service is prepared.